addons/web/static/tests/views/kanban_tests.js

```diff
@@ -2831,6 +2831,39 @@ QUnit.module('Views', {
         kanban.destroy();
     });
 
+    QUnit.test('kanban view with default_group_by and withou groupby menu', async function (assert) {
```

Comment:
without

Reply:
Maybe add a test with search_default targeting a groupby in action context. It is slightly different from your test for the list view below.

Reply:
@Polymorphe57 Added test, would you please check now?